src/components/listing/listing.js

```diff
@@ -8,7 +8,7 @@ const Listing = ({ listingComponent, arrayOfContent, listingClass }) => {
     <ul className={`listing ${listingClass || ''}`}>
       {arrayOfContent.map(props => (
```

Comment:
this `props` and `props.blaah`...

how about this kind of solution?
```suggestion
      {arrayOfContent.map(({ blaah }) => (
```